### PR TITLE
Time range selectors

### PR DESCRIPTION
**Pull request by evulins**
This PR is about the time range selectors:

- Displaying default periods for limiting the timeframe of the graph
- When the time period is changed, changing the scope of the data displayed on the graph and changing the values for min, max and avg
- never disabling buttons,

[Trello task](https://trello.com/c/P9tr2PaU/1829-session-details-default-time-periods)

### CODE REVIEW

File: app/javascript/react/components/Graph/Graph.style.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 import styled from "styled-components";
 import { media } from "../../utils/media";
+import { gray200, red } from "../../assets/styles/colors";
```
Comment (JoannaThelema) [inline app/javascript/react/components/Graph/Graph.style.tsx:3]: leftover?

Comment (bazelajoanna) [inline app/javascript/react/components/Graph/Graph.style.tsx:3]: Can be deleted

File: app/javascript/react/components/Graph/Graph.style.tsx [hunk 2]
```diff
@@ -50,7 +51,6 @@ const Container = styled.div`
       display: none;
       z-index: -1;
     }
-  }
 `;
```
Comment (bazelajoanna) [inline app/javascript/react/components/Graph/Graph.style.tsx:54]: } missing

File: app/javascript/react/components/Graph/graphConfig.ts [hunk 3]
```diff
@@ -290,55 +297,80 @@ const getTooltipOptions = (measurementType: string, unitSymbol: string) => ({
...
+    return {
+      ...baseOptions,
+      buttons: [
+        totalDuration < MILLISECONDS_IN_A_5_MINUTES
+          ? { type: "all", text: "5 MINUTES" }
+          : { type: "minute", count: 5, text: "5 MINUTES" },
+        totalDuration < MILLISECONDS_IN_AN_HOUR
+          ? { type: "all", text: "1 HOUR" }
+          : { type: "minute", count: 60, text: "1 HOUR" },
         { type: "all", text: "ALL" },
```
Comment (JoannaThelema) [inline app/javascript/react/components/Graph/graphConfig.ts:367]: text in translations?

**Review by JoannaThelema — APPROVED** [review]

**Review by bazelajoanna — APPROVED** [review]
🥇

**Pull request merged**